src/components/input-search/input-search.module.css

```diff
@@ -0,0 +1,25 @@
+.input-search {
+  border-radius: 50vh;
+  width: 100%;
+  padding: 6px 35px 6px 12px;
+  font-size: 12px;
+  line-height: 16px;
+  border: 1px solid theme("colors.grey.blue");
+}
+
+.input-search:focus {
+  outline: none;
+}
+
+.input-search-big {
+  border-radius: 50vh;
+  width: 100%;
+  padding: 6px 35px 6px 12px;
+  font-size: 14px;
+  line-height: 20px;
+  border: 1px solid theme("colors.grey.blue");
+}
+
+.input-search-big:focus {
```

Comment:
First we work in BEM methodology, so modifyers should be separated with --
Example: .input-search--big
Second, css modules support nesting, so it should be like this:
```
.input-search {
    ...
    &--big {
    }
    &:focus {
    }
}
```

Reply:
Working on that!

Reply:
I think in BEM traditionally used this style: `some-block__some-element_some-modifier`. Also our webpack configured for transform kebab-case to camelCase in CSS modules

Reply:
fix